File: Python/Practice/LeetCode/Easy/Maximum Product Difference Between Two Pairs.py
# https://leetcode.com/problems/maximum-product-difference-between-two-pairs/

class Solution:
    def maxProductDifference(self, nums: List[int]) -> int:
        nums.sort()
        return (nums[len(nums) -1] * nums[len(nums)-2]) - (nums[0] * nums[1])

File: TypeScript/Practice/LeetCode/Easy/Maximum Product Difference Between Two Pairs.ts
// https://leetcode.com/problems/maximum-product-difference-between-two-pairs/description/

function maxProductDifference(nums: number[]): number {
    nums.sort((a, b) => b - a);
    return (nums[0] * nums[1]) - (nums[nums.length-1] * nums[nums.length-2]);
};